goth_tests/e2e/vm/test_e2e_x509_signature_outbound.py

```diff
@@ -24,24 +24,17 @@
 from goth_tests.helpers.negotiation import DemandBuilder, negotiate_agreements
 from goth_tests.helpers.probe import ProviderProbe
 
-logger = logging.getLogger("goth.test.e2e_outbound")
+logger = logging.getLogger("goth.test.x509_signature_outbound")
 
 
 @pytest.mark.asyncio
-async def test_e2e_outbound(
+async def test_e2e_x509_signature_outbound(
     common_assets: Path,
     default_config: Path,
     config_overrides: List[Override],
     log_dir: Path,
 ):
-    """Test successful flow requesting a outbound curl task with goth REST API client."""
-
-    # Test external api request just one Requestor and one Provider
-    nodes = [
-        {"name": "requestor", "type": "Requestor"},
-        {"name": "provider-1", "type": "VM-Wasm-Provider", "use-proxy": True},
-    ]
-    config_overrides.append(("nodes", nodes))
```

Comment:
Deleted this override because it is unnecessary and makes debugging problems more difficult (nodes config is in `goth-config.yml`).